specification/e-referrals-service-api.yaml

```diff
@@ -315,6 +318,9 @@ paths:
 
         You can check to see what referral letter information has already been added to a referral by using [[HYPERLINK_A005]], [[HYPERLINK_A007]], and [[HYPERLINK_A006]] endpoints where necessary.
 
+        You can generate a letter for the patient by using [[HYPERLINK_A019]]. 
+        The letter summarises the current state of the referral and any steps the patient may need to follow.
+
       summary: A021 - Create referral and send for triage
```

Comment:
I assume these changes have been done "while you;re there"

Reply:
that's right.  It was and ad-hoc request coming form business